bin/create.js

```diff
@@ -154,7 +171,16 @@ function handleCreate (argv) {
         argv.file_name = Path.resolve(process.cwd(), fileOrUrl);
         
         try {
-            argv.code = Fs.readFileSync(argv.file_name, 'utf8');
+            var fileOrDirectory = Fs.lstatSync(argv.file_name);
+
+            if(fileOrDirectory.isFile()) {
+                argv.code = Fs.readFileSync(argv.file_name, 'utf8');
+            } else {
+                argv.code = walk(argv.file_name)
```

Comment:
I think that instead of blindly adding every `.js` file, we should give the user the ability to use glob patterns. This seems to be the way most such interfaces work. As @tjanczuk said, I would also prefer to see the directory adding in a separate PR.